### PR TITLE
[sdks] Fix the ios sdk build.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: sdks/builds/Makefile [hunk 1]
```diff
@@ -6,7 +6,8 @@ RELEASE=$(if $(filter $(CONFIGURATION),release),1)
 
 CCACHE:=$(shell which ccache)
 
-ifneq ($(shell uname),"Darwin")
```
Comment (luhenry) [inline sdks/builds/Makefile]: What's the difference? How is it breaking?

Comment (vargaz) [reply sdks/builds/Makefile]: The original line evals to true on osx, breaking the ios sdk lane. Couldn't figure out why that happens, the new version seems to work in both cases.

**Comment by vargaz** [comment]
So is this ok ?

**Comment by vargaz** [comment]
@monojenkins backport 2018-02

**Comment by vargaz** [comment]
@monojenkins backport 2017-12

**Comment by luhenry** [comment]
Please make sure it's backported to 2018-02 and the backport PR links back to this one.

**Pull request merged**